react-validated-input-component/my-app/src/ValidatedInput.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import './ValidatedInput.css';
+
+export default function ValidatedInput() {
+  const [password, setPassword] = useState('');
+  const [isValid, setIsValid] = useState(false);
+
+  const handleChange = (event) => {
+    const input = event.target.value;
+    setPassword(input);
+    setIsValid(input.length >= 8);
+  };
+
+  return (
+    <div className="validated-input">
+      <label className="password-text">Password</label>
```

Comment:
Make sure your `<label>`s are attached to their form controls, either by wrapping them or via the `for` attribute.

Reply:
Thanks for flagging, it's been adjusted!